instrumentation/grpc-1.5/src/main/java/io/opentelemetry/auto/instrumentation/grpc/client/GrpcClientDecorator.java

```diff
@@ -28,12 +29,18 @@
       OpenTelemetry.getTracerProvider().get("io.opentelemetry.auto.grpc-1.5");
 
   public Span onClose(final Span span, final io.grpc.Status status) {
-
-    span.setAttribute("status.code", status.getCode().name());
-    span.setAttribute("status.description", status.getDescription());
-
-    onError(span, status.getCause());
+    super.onError(span, status.getCause());
```

Comment:
This looks strange to me without first checking if `getCause() != null`

Reply:
Our previous code was doing that too, but I prefer to do null checks at callers too so added